chat-widget/automation_tests/readme.md

```diff
@@ -0,0 +1,30 @@
+# How to run playwright testcases against your environment
```

Comment:
Please modify this file and remove unused document sections

Reply:
Done